lib/Document/Editable/EditableHandler.php

```diff
@@ -342,13 +342,17 @@ protected function buildBrickTemplateReference(TemplateAreabrickInterface $brick
     {
         if ($brick->getTemplateLocation() === TemplateAreabrickInterface::TEMPLATE_LOCATION_BUNDLE) {
             $bundle = $this->bundleLocator->getBundle($brick);
+            $bundleName = $bundle->getName();
+            if ('Bundle' === substr($bundleName, -6)) {
```

Comment:
```suggestion
            if (str_ends_with($bundleName, 'Bundle')) {
```

see https://www.php.net/manual/de/function.str-ends-with.php

Reply:
Thanks! 👍
Just copied it from there: https://github.com/symfony/symfony/blob/5.x/src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php#L176 😁

Reply:
You're welcome. I thought since Pimcore X has the PHP8 requirement, it should be fine to use this method. :)